site_scons/prereq_tools/base.py

```diff
@@ -459,7 +459,7 @@ def __init__(self, env, opts):
                               ['gcc', 'covc', 'clang', 'icc'], ignorecase=2))
         opts.Add(EnumVariable('WARNING_LEVEL', "Set default warning level", 'error',
                               ['warning', 'warn', 'error'], ignorecase=2))
-
+        opts.Add(BoolVariable('ASSERT_ENABLED', 'D_ASSERT macro enabled', False))
```

Comment:
Shouldn't the default be true?  Or at least enabled with dev or debug build?

Reply:
Yes, but there are a lot of compiler issues on various distros that I'm working through first.  Once this PR is stable as-is (and can pass daily-testing) then I'll change some of the GitHub actions builds to set it to False and change the default to True.